homeassistant/components/holiday/config_flow.py

```diff
@@ -0,0 +1,99 @@
+"""Config flow for Holiday integration."""
+from __future__ import annotations
+
+from typing import Any
+
+from babel import Locale
+from holidays import list_supported_countries
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_COUNTRY
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.selector import (
+    CountrySelector,
+    CountrySelectorConfig,
+    SelectSelector,
+    SelectSelectorConfig,
+    SelectSelectorMode,
+)
+
+from .const import CONF_PROVINCE, DOMAIN
+
+SUPPORTED_COUNTRIES = list_supported_countries(include_aliases=False)
+
+
+class HolidayConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Holiday."""
+
+    VERSION = 1
+
+    data: dict[str, Any] = {}
```

Comment:
Please avoid setting a mutable object as class attribute. It's error prone. Move it to an `__init__` method and set it as an instance attribute.